include/bout/paralleltransform.hxx

```diff
@@ -76,6 +76,25 @@ public:
 
   virtual bool canToFromFieldAligned() = 0;
 
+  struct positionsAndWeights {
```

Comment:
`PascalCase` for types, please. I wonder if there's a better, more concise name.

Reply:
Have changed it to PascalCase. I didn't write this bit of code and that was irritating me.